kovri.md

```diff
@@ -1,22 +1,22 @@
 ## 1) - Intro
 
-Monero is secure, untraceable, electronic cash. It is open-source, decentralized, and freely accessible to all. In this video, we will focus on an anonymity technology, called Kovri.
+Monero is secure, untraceable, electronic cash. It is open-source, decentralized, and available for anyone to use. In this video, we will focus on an anonymity technology, called Kovri.
```

Comment:
>available for anyone

NACK, there is a difference between `available for anyone` and `freely available to everyone`. Not all OSS is FOSS.